src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_rate.scss

```diff
@@ -43,15 +43,15 @@ $arrow-width: 1em;
         left: -$arrow-width;
         width: 0;
         height: 0;
-        border-top: $line-height/2 * 1em solid transparent;
-        border-bottom: $line-height/2 * 1em solid transparent;
+        border-top: $line-height/2 * 1em solid $color-structure-transparent;
+        border-bottom: $line-height/2 * 1em solid $color-structure-transparent;
```

Comment:
AFAIK this is a but in scss-lint and is already fixed in the latest master (See https://github.com/causes/scss-lint/blob/master/CHANGELOG.md)